notebook/utils/s1_s2_reader.py

```diff
@@ -41,7 +41,7 @@ def __init__(
         with (Path(s2_input_dir) / "timestamp.pkl").open("rb") as f:
             s2_timesteps = pickle.load(f)
 
-        self.s1_aligned_index = [self.nearest_ind(s1_timesteps, d) for d in s2_timesteps]
+        self.s1_aligned_index = [self.nearest_ind(s2_timesteps, d) for d in s1_timesteps]
```

Comment:
It would be better if alignment was determined by a parameter so we can easily test out one or the other by just using a different argument, what do you think?

Reply:
Ok, will fix.